README.md

```diff
@@ -4,11 +4,73 @@ This repository contains the source code for SKDB and its associated
 components.
 
 
-## Dependencies
+## Running Prepackaged SKDB
 
-You will need at least the following installed:
+You can run the latest release of SKDB's development server with Docker:
 
- * LLVM 15.
- * wasm-ld 15 (this is part of LLVM, but your distribution might split it into
-   a separate package: your wasm-ld version must match your LLVM version).
- * typescript 5.2 or later.
+```sh
+docker run -it -p 3586:8080 skiplabs/skdb-dev-server
```

Comment:
This will need to be updated once https://github.com/SkipLabs/skdb_www/pull/66 lands, right?

Reply:
Indeed.